homeassistant/components/sanix/strings.json

```diff
@@ -0,0 +1,39 @@
+{
+  "config": {
+    "step": {
+      "user": {
+        "description": "To get the Serial number and the Token you just have to sign in to the Sanix Dashboard and open the Help -> System version page. https://sanix.bitcomplex.pl/",
+        "data": {
+          "serial_no": "Serial number",
+          "token": "[%key:common::config_flow::data::access_token%]"
+        }
+      }
+    },
+    "error": {
+      "invalid_auth": "[%key:common::config_flow::error::invalid_auth%]",
+      "unknown": "[%key:common::config_flow::error::unknown%]"
+    },
+    "abort": {
+      "already_configured": "[%key:common::config_flow::abort::already_configured_device%]"
+    }
+  },
+  "entity": {
+    "sensor": {
+      "service_date": {
+        "name": "Service date"
+      },
+      "time": {
+        "name": "Last status update"
+      },
+      "fill_perc": {
+        "name": "Fill percentage"
+      },
```

Comment:
We usually try to avoid having the unit of measurement in the name, so can we maybe name this something like "filled" or "content" or "stored" or whatever